src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -225,6 +243,45 @@ public function load(array $configs, ContainerBuilder $container)
             'Symfony\\Bundle\\FrameworkBundle\\Controller\\Controller',
         ));
 
+        $container->registerForAutoconfiguration(Command::class)
+            ->addTag('console.command');
+        $container->registerForAutoconfiguration(ResourceCheckerInterface::class)
+            ->addTag('config_cache.resource_checker');
+        $container->registerForAutoconfiguration(ServiceSubscriberInterface::class)
+            ->addTag('container.service_subscriber');
+        $container->registerForAutoconfiguration(DataCollectorInterface::class)
+            ->addTag('data_collector');
+        $container->registerForAutoconfiguration(FormTypeInterface::class)
+            ->addTag('form.type');
+        $container->registerForAutoconfiguration(FormTypeExtensionInterface::class)
+            ->addTag('form.type_extension');
```

Comment:
Fixed! Thank you - I was trying to be careful not to select tags like that - obviously missed on. Unrelated, that required attribute is a bummer for the user since it's also in the class itself :)